e2e-tests/playwright/utils/rbac/rbac.ts

```diff
@@ -0,0 +1,106 @@
+import { Page } from '@playwright/test';
+import { UIhelper } from '../UIhelper';
+import {
+  DeleteRolePO,
+  HomePagePO,
+  RoleFormPO,
+  RoleListPO,
+} from '../../support/pageObjects/page-obj';
+import { UIhelperPO } from '../../support/pageObjects/global-obj';
+
+export class Roles {
+  private page: Page;
+
+  constructor(page: Page) {
+    this.page = page;
+  }
+  static getRolesListCellsIdentifier() {
+    const roleName = new RegExp(/^(role|user|group):[a-zA-Z]+\/[\w@*.~-]+$/);
+    const usersAndGroups = new RegExp(
+      /^(1\s(user|group)|[2-9]\s(users|groups))(, (1\s(user|group)|[2-9]\s(users|groups)))?$/,
+    );
+    const permissionPolicies = /\d/;
+    return [roleName, usersAndGroups, permissionPolicies];
+  }
+
+  static getUsersAndGroupsListCellsIdentifier() {
+    const name = new RegExp(/^[a-zA-Z0-9]+(?:-[a-zA-Z0-9]+)*$/);
+    const type = new RegExp(/^(User|Group)$/);
+    const members = /^(-|[0-9]+)$/;
+    return [name, type, members];
+  }
+
+  static getPermissionPoliciesListCellsIdentifier() {
+    const policies =
+      /^(?:(Read|Create|Update|Delete)(?:, (?:Read|Create|Update|Delete))*|Use)$/;
+    return [policies];
+  }
+
+  static getRolesListColumnsText() {
+    return ['Name', 'Users and groups', 'Permission Policies', 'Actions'];
+  }
+
+  static getUsersAndGroupsListColumnsText() {
+    return ['Name', 'Type', 'Members'];
+  }
+
+  static getPermissionPoliciesListColumnsText() {
+    return ['Plugin', 'Permission', 'Policies'];
+  }
+
+  async createRole(name: string, uiHelper: UIhelper) {
```

Comment:
Instead of passing UIHelper as a parameter, can we initialize it like : https://github.com/janus-idp/backstage-showcase/blob/c143d99adcb661b651bd35cf9ecfee6029fe0b48/e2e-tests/playwright/support/pages/Clusters.ts#L10